### PR TITLE
Release/v0.11.0

### PR DESCRIPTION
**Pull request by RatakondalaArun**
- Support for macOS Icons [#407](https://github.com/fluttercommunity/flutter_launcher_icons/pull/407)
- CLI-improvement [#400](https://github.com/fluttercommunity/flutter_launcher_icons/pull/400)
- Add `repository` and `issue_tracker` [#411](https://github.com/fluttercommunity/flutter_launcher_icons/pull/411) (thanks to @patelpathik)
- Fix indent in web/manifest.json [#407](https://github.com/fluttercommunity/flutter_launcher_icons/pull/407)
- Fix the icons 50 and 57 in `contents.json` [#412](https://github.com/fluttercommunity/flutter_launcher_icons/pull/412) (thanks to @adnanjpg)
- Fix typos [#405](https://github.com/fluttercommunity/flutter_launcher_icons/pull/405) (thanks to @edwardmp)
- Added newline to EOF [#325](https://github.com/fluttercommunity/flutter_launcher_icons/pull/325) (thanks to @sandersaelmans)

### CODE REVIEW

**Comment by RatakondalaArun** [comment]
Waiting for #325 to get merged

**Comment by MarkOSullivan94** [comment]
@RatakondalaArun #325 merged now

**Comment by MarkOSullivan94** [comment]
@RatakondalaArun I'd say there's enough changes in this that we could up it to v0.11.0

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ An example is shown below. More complex examples [can be found in the example pr
 
 ```yaml
 dev_dependencies:
-  flutter_launcher_icons: "^0.10.0"
+  flutter_launcher_icons: "^0.10.1"
```
Comment (MarkOSullivan94) [inline README.md:18]:
```suggestion
  flutter_launcher_icons: "^0.11.0"
```

File: pubspec.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ dependencies:
   yaml: ^3.1.0
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
```
Comment (MarkOSullivan94) [inline pubspec.yaml:19]: Why change to `"` instead of "'"?

Comment (RatakondalaArun) [reply pubspec.yaml:19]: I guess it was my auto formatter

**Pull request closed without merging**